log/format.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	timeFormat     = "2006-01-02T15:04:05-0700"
-	termTimeFormat = "01-02|15:04:05"
+	termTimeFormat = "01-02|15:04:05.999999"
```

Comment:
do we really need nanosecond resolution on log stamps?

Reply:
With all the race conditions we had to track and fix, this was very useful.